test/specs/postMH/test.pre.mh.journeys.js

```diff
@@ -43,27 +44,43 @@ describe("Pre-MH journeys when MH is switched on", () => {
   });
 
   it("can create a follow-up claim for a pre-MH sheep review claim if the follow-up visit date is before the MH release date", async () => {
-    const claimNumber = await createSheepReviewClaim(PRE_MULTIPLE_HERD_SBI, {
-      multipleHerdFlag: true,
-      isUnnamedHerdClaimPresent: true,
-    });
+    /* Prerequisites: This test requires an application and a pre-MH sheep review claim, which are inserted using the script
```

Comment:
It would be nice if we could do without these comments - ideally the name of the test and the names of the functions should explain what is happening.

If we want to add a very short comment like // changelog/insert_pre_mh_application_review.sql to point to which SQL file it uses, that would make sense.